modules/iam/users/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "usernames" {
+  description = "Desired names for IAM users"
+  type        = list(string)
+}
+
+variable "has_login" {
+  description = "Whether to create IAM user login profile"
+  type        = bool
+  default     = true
```

Comment:
It doesn't benefit to have console access for bot and it will make additional opportunity for attack.

Just to be safe we can disable bots login and generate bots credentials with using our admin account.

Reply:
Great to know, updated in [216c662](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/216c6628bd63c3a616b795c11beae32c41478579) and [d1c4c1e](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/d1c4c1e01123f1e7a82f2cea6547678b90571995)